tests/ui/pages/desktop/collections.py

```diff
@@ -0,0 +1,29 @@
+from pypom import Region
+from selenium.webdriver.common.by import By
+
+from base import Base
+
+
+class Collections(Base):
+    """ Collections page. """
+
+    _item_locator = (By.CSS_SELECTOR, ".item")
```

Comment:
I think in `addons-server` we are generally striving to use single-quotes in favor of double-quotes for strings. There's old code that uses double-quotes but new code should use single-quotes where possible.